### PR TITLE
Update logos on AI page

### PR DESCRIPTION
**Pull request by carkod**
## Done

Update logos according to [copy docs](https://docs.google.com/document/d/1HP9YdHF11yVQ9SLl0VXhLezDZREsr1GHIKNi9wT1Gv0/edit#heading=h.n4ma01lh3jac)

Logos: https://assets.ubuntu.com/manager?tag=kubeflow-page-logos

## QA

- Go to https://ubuntu-com-13345.demos.haus/ai and check logos are updated according to [copy doc](https://docs.google.com/document/d/1HP9YdHF11yVQ9SLl0VXhLezDZREsr1GHIKNi9wT1Gv0/edit#heading=h.n4ma01lh3jac)
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6585

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/ai/index.html [hunk 1]
```diff
@@ -60,31 +60,34 @@ <h2>Develop artificial intelligence projects on any environment</h2>
...
         <div class="p-logo-section__item">
-          {{ image (
-            url="https://assets.ubuntu.com/v1/5343281c-nvidia-logo.png",
-            alt="Nvidia",
-            width="254",
-            height="256",
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/1652445f-keras-logo.png",
+            alt="",
```
Comment (mtruj013) [inline templates/ai/index.html:66]: We need alt text for all of these

File: templates/ai/index.html [hunk 2]
```diff
@@ -60,31 +60,34 @@ <h2>Develop artificial intelligence projects on any environment</h2>
...
             loading="lazy",
             attrs={"class": "p-logo-section__logo"}
             ) | safe
           }}
         </div>
         <div class="p-logo-section__item">
-          {{ image (
+          {{
+            image (
             url="https://assets.ubuntu.com/v1/3b24973b-lenovo-logo.png",
```
Comment (mtruj013) [inline templates/ai/index.html:91]: This one isn't listed in the doc section

File: templates/ai/index.html [hunk 3]
```diff
@@ -133,16 +139,172 @@ <h2>Develop artificial intelligence projects on any environment</h2>
...
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/c6a757b1-pachyderm-logo.png",
```
Comment (mtruj013) [inline templates/ai/index.html:246]: This one looks squished 
Screenshot from 2023-11-17 12-16-14

File: templates/ai/index.html [hunk 4]
```diff
@@ -133,16 +139,172 @@ <h2>Develop artificial intelligence projects on any environment</h2>
...
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/73accfaf-nvidia-triton-server-logo.png",
```
Comment (mtruj013) [inline templates/ai/index.html:272]: This one looks misaligned a bit 
image

**Review by mtruj013 — DISMISSED** [review]
Thanks @carkod ! A few comments in the code, maybe the [dense logo section](https://vanillaframework.io/docs/patterns/logo-section#dense) would be better here since there are so many, not sure what design instructions have already been given though

**Comment by mtruj013** [comment]
@carkod sorry, forgot to mention this one but the Jupyter logo is missing

File: templates/ai/index.html (hunk 4 above)
Comment (carkod) [reply templates/ai/index.html:272]: I think this is left aligned, if we align this to center than then text would be coming out on the left side.

**Comment by carkod** [comment]
> @carkod sorry, forgot to mention this one but the Jupyter logo is missing

Thanks. All comments addressed except the alignement of Nvidia logo, that was provided by design

**Comment by carkod** [comment]
@mtruj013 can I get another review?

File: templates/ai/index.html [hunk 5]
```diff
@@ -109,9 +139,22 @@ <h2>Develop artificial intelligence projects on any environment</h2>
...
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/7cf825b5-opensearch-logo.png",
             alt="",
```
Comment (mtruj013) [inline templates/ai/index.html:156]:
```suggestion
            alt="OpenSearch",
```

**Review by mtruj013 — APPROVED** [review]
LGTM!

**Pull request merged**